package.json

```diff
@@ -22,11 +22,9 @@
     "build": "preconstruct build",
     "build:watch": "preconstruct watch",
     "changeset": "changeset",
-    "version-packages": "changeset version",
-    "prerelease": "cross-env NODE_ENV=production yarn auth && yarn build",
+    "changeset:version-and-format": "changeset version && prettier --write --parser json '**/package.json' && yarn install",
```

Comment:
Note the `yarn install`. This is needed now because `yarn.lock` contains the bumped versions.

<img width="1115" alt="image" src="https://user-images.githubusercontent.com/1110551/140609726-a9275f27-d1f5-4289-be46-4a601b4bac8c.png">